lib/course_planner/events/event_encoder.ex

```diff
@@ -0,0 +1,19 @@
+defimpl Poison.Encoder, for: CoursePlanner.Events.Event do
```

Comment:
what is the user of this code? i am not familiar

Reply:
It's an Elixir Protocol. `defimpl` overrides the standard encoder (that tries to encode everything inside the `Event` struct). I was having some trouble with the `users` relationship.